package.json

```diff
@@ -12,9 +12,9 @@
     "start": "npm run build -- -w",
     "build": "cross-env NODE_ENV=production babel src -d dist --ignore 'src/**/*.test.js'",
     "test:all": "npm run test:coverage && npm run test:lint",
-    "test": "jest",
-    "test:coverage": "jest --collectCoverageFrom='src/**/*.js' --coverage",
-    "test:watch": "jest --watch",
+    "test": "jest --env node",
```

Comment:
Jest default configuration is `--env = jsdom`, it can simulate the browser environment, I worry that it takes up extra memory